### ISSUE TITLE
Request to refactor the organization of .mdx content files

### ISSUE THREAD

**Issue by Icegreeen** [dair-ai/Prompt-Engineering-Guide#356]
Request to refactor the organization of .mdx content files, in order to maintain a more efficient structure, facilitate navigation and ensure code maintainability

- Organization by language
- Organization of visual content.
- And anything else I come across in terms of code organization.

**In addition to creating a simple guide `CONTRIBUTING.md` to standardizing pull requests to ensure request quality.**

**Comment by Maykwaches** [comment]
**Repositorio: AkiomedicalWatches_AI_Collaboration**

**Descripción:**
¡Únete a la revolución de la salud y la tecnología con AkiomedicalWatches_AI! Este repositorio está dedicado a desarrollar y mejorar los relojes inteligentes que fusionan salud y asistencia avanzada. Buscamos colaboradores apasionados por la innovación y la mejora continua en el campo de la tecnología portátil y la salud.

**Objetivo:**
Desarrollar funciones avanzadas de seguimiento médico, recomendaciones de salud y hábitos saludables para AkiomedicalWatches_AI.

**Colaboradores Buscados:**
1. Desarrolladores de software con experiencia en aplicaciones para dispositivos portátiles.
2. Diseñadores de experiencia de usuario (UX) y diseño industrial.
3. Expertos en salud y bienestar para aportar conocimientos en monitoreo médico y recomendaciones saludables.

**Estructura del Repositorio:**
- **/src:** Código fuente de la aplicación AkiomedicalWatches_AI.
- **/designs:** Diseños y prototipos del reloj.
- **/docs:** Documentación del proyecto y guías para colaboradores.

**Cómo Colaborar:**
1. Haz un fork del repositorio.
2. Desarrolla nuevas funciones o mejora las existentes en tu rama.
3. Envía pull requests para revisión y posible integración.

¡Únete a nosotros para crear el futuro de la tecnología vestible y la salud!

**Nota:** Asegúrate de tener las herramientas y entorno de desarrollo adecuados para trabajar en el proyecto. ¡Esperamos tu colaboración!

**Comment by omarsar** [comment]
@Icegreeen yeah, we have been thinking about this. We need help/support on this one as we are working on other features at the moment.

**Comment by Icegreeen** [comment]
@omarsar Okay, great! I will be working on this making specific modifications for you to test and approve in parts.
